benchmarks/benchmark_functions/default_settings.py

```diff
@@ -49,3 +54,49 @@ def Template(params_):
 		template = Template
 
 	return device, diff_method, interface, params, template, measurement
+
+
+def _vqe_defaults(hyperparams):
+	"""Uses hyperparameters or defaults to construct the components of the VQE circuit for the
+	hydrogen molecule with the sto-3g basis set.
+
+	Args:
+		hyperparams (dict): hyperparameters provided by user
+	"""
+
+	H_coeffs = np.array([-0.05963862, 0.17575739, 0.17575739, -0.23666489, -0.23666489,
+						  0.17001485, 0.04491735, -0.04491735, -0.04491735, 0.04491735,
+						  0.12222641, 0.16714376, 0.16714376, 0.12222641, 0.17570278])
+
+	H_ops = [Identity(wires=[0]), PauliZ(wires=[0]), PauliZ(wires=[1]), PauliZ(wires=[2]),
+			 PauliZ(wires=[3]), PauliZ(wires=[0]) @ PauliZ(wires=[1]),
+			 PauliY(wires=[0]) @ PauliX(wires=[1]) @ PauliX(wires=[2]) @ PauliY(wires=[3]),
+			 PauliY(wires=[0]) @ PauliY(wires=[1]) @ PauliX(wires=[2]) @ PauliX(wires=[3]),
+			 PauliX(wires=[0]) @ PauliX(wires=[1]) @ PauliY(wires=[2]) @ PauliY(wires=[3]),
+			 PauliX(wires=[0]) @ PauliY(wires=[1]) @ PauliY(wires=[2]) @ PauliX(wires=[3]),
+			 PauliZ(wires=[0]) @ PauliZ(wires=[2]), PauliZ(wires=[0]) @ PauliZ(wires=[3]),
+			 PauliZ(wires=[1]) @ PauliZ(wires=[2]), PauliZ(wires=[1]) @ PauliZ(wires=[3]),
+			 PauliZ(wires=[2]) @ PauliZ(wires=[3])]
+
+	electrons = 2
+	qubits = 4
+
+	singles, doubles = qchem.excitations(2, 4, delta_sz=0)
+	s_wires, d_wires = qchem.excitations_to_wires(singles, doubles)
+	hf_state = qchem.hf_state(electrons, qubits)
+	ansatz = partial(UCCSD, init_state=hf_state, s_wires=s_wires, d_wires=d_wires)
+
+	Hamiltonian = hyperparams.pop('Hamiltonian', qml.Hamiltonian(H_coeffs, H_ops))
+	ansatz = hyperparams.pop('template', ansatz)
+	params = hyperparams.pop('params', np.random.normal(0, np.pi, len(singles) + len(doubles)))
+	n_steps = hyperparams.pop('n_steps', 1)
+	device = hyperparams.pop('device', 'default.qubit')
+	interface = hyperparams.pop('interface', 'autograd')
+	diff_method = hyperparams.pop('diff_method', 'best')
+	optimize = hyperparams.pop('optimize', True)
```

Comment:
I guess it's here: if we set `optimize=True` then let's make sure the bug with grouping is fixed!